src/consensus/aft/raft.h

```diff
@@ -980,6 +994,7 @@ namespace aft
       const uint8_t* data,
       size_t size)
     {
+      LOG_DEBUG_FMT("Received evidence for view:{}, from {}", r.view, from);
```

Comment:
Seems weird that we sometimes ":{}" and sometimes " {}" for values. I don't really have a preference, but I would like it if we stuck to one style.